qcodes/instrument_drivers/american_magnetics/AMI430.py

```diff
@@ -28,32 +30,42 @@ class AMI430(IPInstrument):
         current_ramp_limit (float): current ramp limit in ampere per second
         persistent_switch (bool): whether this magnet has a persistent switch
     """
-
     mocker_class = MockAMI430
     default_current_ramp_limit = 0.06  # [A/s]
 
     def __init__(self, name, address=None, port=None, persistent_switch=True,
-                 reset=False, current_ramp_limit=None, terminator='\r\n', testing=False, **kwargs):
+                 reset=False, current_ramp_limit=None,
+                 terminator='\r\n', testing=False, **kwargs):
+
+        print('Debug: {}, {}'.format(address, port))
```

Comment:
remove?

Reply:
Yeah that should be removed.

Reply:
I don't know, maybe this should be removed?